test/unit/package-lock_prettify.cmake

```diff
@@ -0,0 +1,67 @@
+
```

Comment:
I'd prefer a more simple unit test of the output of `cpm_prettyfy_package_arguments()` here, as the integrated functionality is already tested. Optimally with multiple input examples.